pkg/operation/garden/garden.go

```diff
@@ -253,6 +253,12 @@ func ReadGardenSecrets(ctx context.Context, c client.Reader, namespace string, l
 			secretsMap[v1beta1constants.GardenRoleGlobalShootRemoteWriteMonitoring] = &monitoringSecret
 			logInfo = append(logInfo, fmt.Sprintf("monitoring basic auth secret %q", secret.Name))
 		}
+
+		if secret.Labels[v1beta1constants.GardenRole] == v1beta1constants.GardenRoleGlobalShootExternalBlackboxExporterMonitoring {
```

Comment:
maybe add a comment above this block to make it consistent with the other similar blocks